sdk/rpc/server/server.go

```diff
@@ -117,6 +118,20 @@ func (t *RPCServer) ExecutableNeedsAuth(req proto.ExecutableNeedsAuthRequest, re
 // CredentialImport is a remote version of the Import() function in schema.CredentialType.
 // The call is forwarded to the Import() function of the credential identified by req.CredentialID.
 func (t *RPCServer) CredentialImport(req proto.ImportCredentialRequest, resp *sdk.ImportOutput) error {
+	defer func() {
+		if err := recover(); err != nil {
+			caughtPanic := fmt.Errorf("your locally built plugin's importing failed with the following panic: %s; and with stack trace: %s", err, string(debug.Stack()))
+			diagnostics := sdk.Diagnostics{Errors: []sdk.Error{{Message: caughtPanic.Error()}}}
```

Comment:
Can we pull this into a separate function? I think the CLI gives enough context to return a more generic error, like:

```
locally built plugin panicked: %s\nstack trace:\n%s
```